### DISCUSSION TITLE
Why Piped was down but InnerTune is still working okey? How to?

### DISCUSSION THREAD

**Discussion by maxrave-dev** [z-huang/music#974] [Q&A] [answered]
I am trying to create a music app like yours, I see your app using PipedApi to get stream data. By the way, all Piped Instance is broken down (403 Error). But I use your app, this is still working. I don't know why. Can you explain it for me? Thanks @z-huang

**Comment by owenconnorz** [comment, marked as answer]
He uses innertube api as default piped was just away to get age restriction on songs  bypass @maxrave-dev
